### PR TITLE
*  add debug config and download location mount to docker config

### PR DESCRIPTION
**Pull request by vianvi**

### CODE REVIEW

**Review by timja — DISMISSED** [review]

File: tests/amountClaim/pages/submitted.js [hunk 1]
```diff
@@ -29,5 +29,6 @@ module.exports = {
     I.see('1. Download the sealed claim form (this will open in a new window).')
     I.scrollPageToBottom()
     I.click('Download the sealed claim form')
+    I.wait(10)
```
Comment (timja) [inline tests/amountClaim/pages/submitted.js]: can't you do something better than this?
Wait till the request completes? Check for existence of the file maybe?
This adds 10 seconds on to every test that submits a claim

File: test-data.js [hunk 2]
```diff
@@ -4,3 +4,21 @@ exports.user = {
   email: email,
   password: 'Password123'
 }
+
+exports.submitPageData = {
+  downloadPDFLink: 'Download the sealed claim form',
+  claimIssuedText: 'Your claim has been issued',
+  emailConfirmation: 'We\'ve emailed confirmation to: ',
+  feesPaid: 'Fee paid: £455',
+  followSteps: 'Follow these steps to serve a claim:',
```
Comment (timja) [inline test-data.js]: I don't think this is a good approach to be verifying this sort of static data on the page.
It ties the implementation too closely to the tests and means that content changes will need test changes for things that shouldn't need it, which makes it harder for content designers to update tests.

Anything calculated makes sense to do here like the fee, but static text more than a heading should not be verified in my opinion, 
@pragnesh-patel  or @hmcts/cmc any other thoughts?

Comment (pragnesh-patel) [reply test-data.js]: I agree checking static data on page text is prone to making tests brittle, if the text changes.
Also not gd practice in hard-coding values such as fee paid directly in SubmitPageData, It should come from another object in TestData such as export.fees or export.claim

Comment (vianvi) [reply test-data.js]: If we don't verify the text how do we ensure that the correct text has been displayed on page?

Comment (vianvi) [reply test-data.js]: the other problem is as we are not inside the main project - we have to write text here- otherwise one content file should take care for tests as well.No need to maintain separate

Comment (timja) [reply test-data.js]: the text is static, the only reason for it to change is if there was a decision to change it.
No need to test static text in general IMO.

having it in the same project wouldn't help as all text goes directly on the page.
Content files are not great practice as it makes it hard to follow what is actually going on the  page

Comment (vianvi) [reply test-data.js]: I agree with you Tim, however we have to verify the text - if we are checking in unit level - will remove them from here .The one problem is  If we don't verify the text -incase if some one while updating mistakenly updated another key(I am assuming we use Key Value pairs) or completely miss - we are going to see wrong text on page or we are going to miss the content

Comment (timja) [reply test-data.js]: there's no keys to be updated the text is on the page.
any change goes through codereview.

File: test-data.js [hunk 3]
```diff
@@ -4,3 +4,21 @@ exports.user = {
   email: email,
   password: 'Password123'
 }
+
+exports.submitPageData = {
```
Comment (timja) [inline test-data.js:8]: blocking this PR with the static text verification in it

**Review by timja — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 4]
```diff
@@ -27,6 +27,8 @@ services:
         - ./output:/usr/src/app/output
     remote-webdriver:
       image: docker.artifactory.reform.hmcts.net/selenium/standalone-chrome:3.5.2
+      volumes:
+        - ./download:/home/seluser/Downloads
```
Comment (timja) [inline docker-compose.yml]: remove

File: helpers/downloadPdfHelper.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+'use strict'
```
Comment (timja) [inline helpers/downloadPdfHelper.js:1]: can't comment on the file but delete the download directory

**Review by timja — APPROVED** [review]
